README.md

```diff
@@ -1,65 +1,107 @@
 
 The easiest way to deploy Ghost is with our official **[Ghost(Pro)](https://ghost.org/pricing/)** managed service. You can have a fresh instance up and running in a couple of clicks with a worldwide CDN, backups, security and maintenance all done for you.
 
-Not only will it save you hours of maintenance per month, but all revenue goes to the Ghost Foundation, which funds the maintenance and further development of Ghost itself. So you’ll be supporting open source software *and* getting a great service **at the same time**! Talk about win/win. :trophy:
+Not only will it save you hours of maintenance per month, but all revenue goes to the Ghost Foundation, which funds the maintenance and further development of Ghost itself. So you’ll be supporting open source software *and* getting a great service! Alternatively if you'd like to support us, we're very grateful to all our backers on [Open Collective](https://opencollective.com/ghost) :heart:
 
-## Self-Hosters
+&nbsp;
 
-Other options are also available if you prefer playing around with servers by yourself, of course. The freedom of choice is in your hands.
+# Quickstart Install
 
-- [Self-hosting Guide](https://docs.ghost.org/v1/docs/getting-started-guide)
+If you want to run your own instance of Ghost, in most cases the best way is to use our **CLI tool**
 
+```
+$ npm install ghost-cli -g
+```
 
-# Theme Developers
+&nbsp;
 
-If you are developing a Ghost theme for your own site or creating themes for others to use we recommend installing Ghost on your own local machine. Luckily we have a brand new Ghost CLI to make this really easy 😄
+Then, if installing locally add the `local` flag to get up an running in under a minute - [Local install docs](https://docs.ghost.org/docs/install-local)
 
-- [Installing Ghost via the CLI](https://docs.ghost.org/v1/docs/install-local)
-- [Theme Developer Docs](https://themes.ghost.org/v1.23.0/)
+```
+$ ghost install local
+```
 
+&nbsp;
 
-# Contributors & Advanced Developers
+or on a server run the full install, including automatic SSL setup using LetsEncrypt - [Production install docs](https://docs.ghost.org/docs/install)
 
-For anyone wishing to contribute to Ghost or to hack/customise core files we recommend following our development setup guides:
+```
+$ ghost install
+```
 
-- [General Contributor Guide](https://docs.ghost.org/v1/docs/contributing)
-- [Developer Setup Instructions](https://docs.ghost.org/v1/docs/working-with-ghost)
-- [Admin Client development guide](https://docs.ghost.org/v1/docs/working-with-the-admin-client)
+&nbsp;
 
+Check out our [official documentation](https://docs.ghost.org/) for more information about [recommended hosting stack](https://docs.ghost.org/docs/hosting), properly [upgrading Ghost](https://docs.ghost.org/v1/docs/upgrade), or want to or would like to learn about developing your own Ghost [themes](https://themes.ghost.org/docs) or working with [the API](https://api.ghost.org).
```

Comment:
> , or want to or would like to learn about developing your own ghost themes or working with the API

This sounds weird to me, although I'm not exactly sure why / what would be a better way 🤔 😬